### DISCUSSION TITLE
RegisterVideoEffect in single stream

### DISCUSSION THREAD

**Discussion by kosmokoks** [HaishinKit/HaishinKit.swift#1573] [Q&A]
I'm connecting to two servers at the same time.
is there a way to add an effect to only one stream, so that it only shows up on the first server, and the other one has no effect?

```
currentEffect = Watermark()
await mixer.screen.registerVideoEffect(currentEffect!)

let stream1 = await netStreamSwitcher.stream
await mixer.addOutput(stream1!)
await mixer.addOutput(previewView!)

let stream2 = await netStreamSwitcher.stream2
await mixer.addOutput(stream2!)
```

**Comment by shogo4405** [comment]
Please stop reposting similar content. It's disruptive. It can't be done, and there are no plans for it.

**Reply by kosmokoks** [reply]
I'm very sorry, 
I just want to know if it's even possible to do my example, i.e. send the stream to two servers, one with some effects and the other without, if it's possible using this library I'd be happy to arrange an e-mail consultation with you

**Reply by shogo4405** [reply]
As I mentioned above, it's not possible.
